sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6999,4 +6999,14 @@ ALTER TABLE country ADD CONSTRAINT fk_country_subcontinent_id FOREIGN KEY (subco
 
 INSERT INTO schema_version (version_number, comment) VALUES (352, '2020-03-17 Create continent and subcontinent #4775');
 
+-- 2020-03-19 Add continent and subcontinent to location #4777
+
+ALTER TABLE location ADD COLUMN continent_id BIGINT;
```

Comment:
Columns also need to be added to the history tables (foreign key isn't necessary there though)